pkg/util/validation/validate.go

```diff
@@ -121,7 +120,6 @@ type SampleValidationMetrics struct {
 	labelNameTooLong       *prometheus.CounterVec
 	labelValueTooLong      *prometheus.CounterVec
 	duplicateLabelNames    *prometheus.CounterVec
-	labelsNotSorted        *prometheus.CounterVec
```

Comment:
Why is this metric no longer relevant? I could use some context :)

Reply:
It's not tracked anymore since https://github.com/grafana/mimir/pull/3640